src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -557,7 +557,9 @@ class GoogleExtensions(
         groupOption <- googleDirectoryDAO.getGoogleGroup(allUsersGroupEmail)
       } yield {
         groupOption match {
+          // $COVERAGE-OFF$Disabling highlighting by default because this confuses coveralls.
           case Some(_) => OkStatus
+          // $COVERAGE-ON$
```

Comment:
Why does the failure None case have consistent test coverage but not the Some(_)? Would we better off adding test coverage for a healthy status check instead of ignoring this line?

Reply:
I don't think we test the whole thing. I print the stackstace, seems the none is covered during some test steup. My understanding: This line automatically happens from HealthMorinitor when receives a call.  The user here is GROUP_All_Users@dev.test.firecloud.org.

Do we want to add test in this case?

Reply:
I'll defer to Doug, but it doesn't seem to be worthwhile to ignore a line from test coverage just because we're not testing it reliably. It also feels like we shouldn't care to be notified if a single line is or isn't being tested, which involves adjusting the threshold for notifications, per Doug's comment earlier.
I'd say either test the line or don't care that it's not being tested. A special exemption doesn't seem worthwhile. At the very least, comment why we think disabling the line is the right thing.

Reply:
agreed, we should not commit this change.